.craft.yml

```
@@ -0,0 +1,23 @@
+minVersion: 1.0.0
+changelogPolicy: auto
+
+artifactProvider:
+  name: none
+statusProvider:
+  name: github
+  config:
+    contexts:
+      - 'build-uptime-checker (sentryio)'
+
+targets:
+  - name: github
+  - id: release
+    name: docker
+    source: ghcr.io/getsentry/uptime-checker
```

Different than other repos, I guess using GHCR as the default source is a better thing to do since it is guaranteed open source and guaranteed not to be tampered with.

---

I think this should be done in 2 separate steps as currently we build on Google Cloud and the GitHub context you block on (defined above) is the Google Cloud Build status anyway. Let's go with that image first and then do the switch over across all repos.

```suggestion
    source: us-central1-docker.pkg.dev/sentryio/uptime-checker/image
```

---

@BYK How can we guarantee if the image on the GCR is being successfully built?

---

@aldy505 that's what Craft does with the status check? GitHub status check would fail if that Google Cloud build fails which then would be picked up by Craft, cancelling the publishing?

---

Ah! TIL.